page.codeberg.gwync.glerm.yml

```diff
@@ -0,0 +1,30 @@
+id: page.codeberg.gwync.glerm
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: glerm
+finish-args:
+  - --filesystem=home
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - pypi-deps.json
+  - name: glerm
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 page.codeberg.gwync.glerm.metainfo.xml /app/share/metainfo/page.codeberg.gwync.glerm.metainfo.xml
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - install icon.png /app/share/icons/hicolor/256x256/apps/page.codeberg.gwync.glerm.png
+      - mkdir -p /app/share/applications/
+      - install -Dm 644 page.codeberg.gwync.glerm.desktop /app/share/applications/page.codeberg.gwync.glerm.desktop
+    sources:
+      - type: file
+        path: page.codeberg.gwync.glerm.metainfo.xml
+      - type: file
+        path: page.codeberg.gwync.glerm.desktop
+      - type: file
+        path: icon.png
```

Comment:
these metadata should be in the upstream repository.

Reply:
They will be. Can they be referred to by URL in build-commands?

Reply:
Yea `path:` -> `url:` and add `sha256:`, make sure url is pinned to a commit so that hash does not change.

Reply:
I'm working on this but the downloaded sha256 keeps changing:
` type: file
        url: https://codeberg.org/gwync/glerm/src/commit/094208e70ca26d9a654e7a43037c070984458648/page.codeberg.gwync.glerm.metainfo.xml
        dest-filename: page.codeberg.gwync.glerm.metainfo.xml
        sha256: 2d470694bc1d510865842202a60731cebc804d6f96ef46c3d63a64dd985e4571
`
The local sha256sum is d81223ef3a2984284619b27c4e4591962c47d58dd841992111c4c699e838d19c

Reply:
You have to use the /raw link otherwise it's an html page https://codeberg.org/gwync/glerm/raw/commit/094208e70ca26d9a654e7a43037c070984458648/page.codeberg.gwync.glerm.metainfo.xml

Reply:
That did it, thanks!